### DISCUSSION TITLE
Export from qmd to rmd or ipynb

### DISCUSSION THREAD

**Discussion by danieltomasz** [quarto-dev/quarto-cli#621] [Q&A] [answered]
Does Quarto support export to rmd or ipnb? I suppose this is possible  via pandoc and command line now, but it is not possible via Rstudio for example?

The motivation for this question is for sharing code with people who doesn't use quarto.

**Comment by jjallaire** [comment]
Via the command line you can do `quarto convert document.qmd` to convert to ipynb.

To "export" to Rmd you can just rename the file to ".Rmd" (and make sure you aren't using Qmd specific features)

**Comment by danieltomasz** [comment]
First of all thank you for developing quarto (playing with it gives me new  strength to work on old projects ;)

I would like to have an option to use quarto  features and then strip these features to create a normal Rmd. 
Use case: I am working on personal computer  and develop notebook using qarto and my corporate environment were  I want to run it too have older version of Rstudio/or doesnt vet easily new binaries.
In that case conversion qmd -> ipynb -> rmd will work I think ? (via `jupytext` for example)
I saw a long discussion on jupytext github regarding  support for  quarto.

I would be nice to have this stripping  export option via quarto too.

**Reply by jjallaire** [reply, marked as answer]
You can actually use the Rmd file extension with Quarto if you want to. So a single Rmd file could be processed by both Quarto and rmarkdown (you'd just have a `format` options key for Quarto and an `output` options key for R Markdown. Quarto features like advanced figure/page layout wouldn't work in Rmd but they also wouldn't break (note that both Quarto and R Markdown support YAML cell options).
